### PR TITLE
chore: fix comment about tuple store types

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: src/compile/selection/transforms/project.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const TUPLE_FIELDS = '_tuple_fields';
 
 /**
  * Do the selection tuples hold enumerated or ranged values for a field?
- * Ranged values can be left-right inclusive (R) or left-inclusive, right-exclusive (R-LE).
+ * Ranged values can be exclusive (E),  left-right inclusive (R) or left-inclusive, right-exclusive (R-RE).
```
Comment (kanitw) [inline src/compile/selection/transforms/project.ts:14]: > Ranged values can be exclusive (E)

- E = enumerate according to the line above
- Enumerate is not range

Comment (domoritz) [reply src/compile/selection/transforms/project.ts:14]: So 
* E = enumerated
* R = ranged but left-right left-right inclusive
* R-RE = ranged left-inclusive, right-inclusive

Comment (kanitw) [reply src/compile/selection/transforms/project.ts]: I think so, but @arvind would know better.

**Comment by domoritz** [comment]
Merging. @arvind please let us know if we misunderstood the comment.

**Pull request merged**